root/dashboard/swag-f2b.py

```diff
@@ -1,8 +1,12 @@
 import json
 import sqlite3
+import sys
 
+f2bdb = "/config/fail2ban/fail2ban.sqlite3"
```

Comment:
Please use an environment variable and combine the results of swag's f2b with the custom f2b, there's no need to display the same section twice

Reply:
(And it won't require any changes to the php)